README.md

```diff
@@ -51,25 +51,29 @@ pnpm cdk ls
 Example output:
 
 ```sh
-
 OrcaBusStatelessWorkflowManagerStack
 OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusBeta/WorkflowManagerStack (OrcaBusBeta-WorkflowManagerStack)
 OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusGamma/WorkflowManagerStack (OrcaBusGamma-WorkflowManagerStack)
 OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusProd/WorkflowManagerStack (OrcaBusProd-WorkflowManagerStack)
 ```
 
-To build the CICD pipeline for workflow manager
+To deploy the CICD pipeline for the workflow manager, switch to the AWS Bastion/Toolchain account (e.g. `export AWS_PROFILE=umccr-bastion-admin`) and then run:
 ```sh
+pnpm cdk synth -e OrcaBusStatelessWorkflowManagerStack
+pnpm cdk diff -e OrcaBusStatelessWorkflowManagerStack
 pnpm cdk deploy -e OrcaBusStatelessWorkflowManagerStack
 ```
 
-To build (test) in the dev account
+To deploy the app, switch to the AWS DEV account (e.g. `export AWS_PROFILE=umccr-dev-admin`) and then run:
 ```sh
 pnpm cdk synth -e OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusBeta/WorkflowManagerStack
 pnpm cdk diff -e OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusBeta/WorkflowManagerStack
 pnpm cdk deploy -e OrcaBusStatelessWorkflowManagerStack/DeploymentPipeline/OrcaBusBeta/WorkflowManagerStack
 ```
 
+> NOTE:
+> While directly pushing to the AWS DEV account is possible, however, we encourage you to make use of automated CICD CodePipeline for most cases. Pushing direct to the AWS PROD account is discouraged as it will bypass necessary test harness and other safety measures. Please check with the team on this in `#orcabus` if a need arises.
```

Comment:
Not sure I would even mention that direct pushes to PROD are possible...
First, that should not happen and second, we put links to our repos into the NATA docs where we claim everything is controlled and automated for PROD. 
I don't think any reviewers will actually read our docs, but if they do it would not look too good.

Reply:
Ok, I will remove PROD part and re-word.

Reply:
Can you check it again now?
https://github.com/OrcaBus/service-workflow-manager/pull/5/commits/491206be965bec4cb525ce63225f8b6a2552df1b

Reply:
Yup, that's perfect!